doc/sphinx/azhelpgen/doc_source_map.json

```diff
@@ -1,5 +1,6 @@
 {
     "az": "src/azure-cli/azure/cli/command_modules/profile/_help.py",
+    "confige": "src/azure-cli/azure/cli/command_modules/config/_help.py",
```

Comment:
extra `e`?

Reply:
Oh! Nice catch. Perhaps our CI should check it? @haroldrandom @MyronFanQiu